app/assets/stylesheets/modal.scss

```diff
@@ -38,16 +37,13 @@ $modal-margin: 12px;
         @media screen and (min-width: $modal-width + 2 * $modal-margin) {
             width: $modal-width;
             position: fixed;
-            top: 50%;
+            top: 48px;
             left: 50%;
-            margin-left: -325px;
-            margin-top: -160px;
+            margin-left: -$modal-width / 2;
         }
 
         @media screen and (max-width: $modal-width + 2 * $modal-margin) {
             position: fixed;
-            margin-top: -160px;
```

Comment:
This made the modal appear off the top of the screen on mobile devices, I've removed it.